appholder/src/main/java/com/android/mdl/app/transfer/TransferManager.kt

```diff
@@ -281,7 +281,10 @@ class TransferManager private constructor(private val context: Context) {
     }
 
     fun sendResponse(deviceResponse: ByteArray) {
-        presentation?.sendDeviceResponse(deviceResponse)
+        presentation?.sendDeviceResponse(deviceResponse,
+                                         { progress ->
```

Comment:
Should be `progress, max` or something, right?

This is failing presubmit checks.